src/core/jsonld.js

```diff
@@ -0,0 +1,101 @@
+// Module core/jsonld
+// Create script element with JSON-LD for SEO document description
+export const name = "core/jsonld";
+
+export function run(conf, doc, cb) {
+  if (!conf.doJsonLd) {
+    return cb();
+  }
+  // Content for JSON
+  const types = ["TechArticle"];
+  if (conf.rdfStatus) types.push(conf.rdfStatus);
+
+  const jsonld = {
+    "@context": ["http://schema.org", {
+      "@vocab": "http://schema.org/",
+      "@language": "en",
+      "w3p": "http://www.w3.org/2001/02pd/rec54#",
+      "foaf": "http://xmlns.com/foaf/0.1/",
+      "schema": "http://schema.org/",
+      "xhv": "http://www.w3.org/1999/xhtml/vocab#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#",
+      "citation": {"@type": "@id"},
+      "contributor": {"@type": "@id"},
+      "datePublished": {"@type": "xsd:date"},
+      "dependencies": {"@type": "@id"},
+      "editor": {"@type": "@id"},
+      "hasPart": {"@type": "@id"},
+      "inLanguage": {"@language": null},
+      "isBasedOn": {"@type": "@id"},
+      "license": {"@type": "@id"},
+      "license": {"@type": "@id"},
+      "worksFor": {"@type": "@id"}
+    }],
+    id: (conf.canonicalURI || conf.thisVersion),
+    type: types,
+    name: conf.title,
+    inLanguage: ($('html', doc).attr('lang') || 'en'),
```

Comment:
We are trying to remove jQuery dependency, so I would use `doc.documentElement.getAttribute("lang")`. (Double quotes are preferred)